packages/core/EventManager.ts

```diff
@@ -466,13 +621,9 @@ export default class EventManager {
         (credential) => credential.id === event.conferenceCredentialId
       );
     } else {
-      videoCredential = this.videoCredentials
-        // Whenever a new video connection is added, latest credentials are added with the highest ID.
-        // Because you can't rely on having them in the highest first order here, ensure this by sorting in DESC order
-        .sort((a, b) => {
```

Comment:
Sort does the sorting in place, so it modifies `this.videoCredentials` itself. Moved this sorting to constructor to make it clear.